website/docs/deploy/configuring-unleash.md

```diff
@@ -101,6 +101,18 @@ unleash.start(unleashOptions);
     - `demo` - Only requires an email to sign-in (was default in v3)
   - customAuthHandler: (function) - custom express middleware handling authentication. Used when type is set to `custom`
   - createAdminUser: (boolean) - whether to create an admin user with default password - Defaults to `true`
+  - initApiTokens: (ApiTokens[]) - Array of Api Tokens to create on startup. Will only be applied if there does not exist any API tokens in the database from before. 
+      Example:
+     ```ts
+     [{
+        environment: '*',
+        project: '*',
+        secret: '*:*:964a287e1b728cb5f4f3e0120df92cb5',
+        type: ApiTokenType.ADMIN,
+        username: 'some-user',
+    }]
```

Comment:
The secret is the thing actually shared with the API client, thus duplicated. By prefixing it with project and environment it gives context, but we currently only support that for "Client Tokens", while admin tokens must be wild-carded.

Reply:
fyi: there is not actual requirements that the secret itself is prefixed, but we do that for new tokens to add context to the user of the token.

Reply:
(because the holder of an API token secret would be able to use the token and that way discover its capabilites, and you should take good care of it, I see no harm in adding the context in to the secret itself).

Reply:
I think I see what you mean. I was just thinking that it seems redundant and that you can easily supply unmatching values unless there is a check. But that's on the software design side, and you certainly have more insight into that than I do, so I trust you've taken these things into consideration. Thanks for the explanation! 🙏🏼

Reply:
When creating these from the API/UI, which is the normal user-flow, you will not be able to set the secret yourself. This is very specific way of short-circuit it all, and thus in find it OK to not build to much additional logic for this scenario.